### PR TITLE
watson 1.3.1 (new formula)

### PR DESCRIPTION
**Pull request by jmaupetit**
Watson is here to help you manage your time. You want to know how much time you are spending on your projects? You want to generate a nice report for your client? Watson is here for you.

### CODE REVIEW

File: Library/Formula/watson.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+class Watson < Formula
+  desc "The wonderful CLI to track your time!"
```
Comment (bfontaine) [inline Library/Formula/watson.rb]: We might want to use something like `"Command-line tool to track your time"`

File: Library/Formula/watson.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/watson", "--help"
```
Comment (bfontaine) [inline Library/Formula/watson.rb]: Please write a test that does something more substantial than e.g. `--help` or `--version`.

**Comment by jmaupetit** [comment]
Thank you for your feedback! I changed what you requested. Do I need to squash my commits?

File: Library/Formula/watson.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
+class Watson < Formula
+  desc "Command-line too to track (your) time"
```
Comment (DomT4) [inline Library/Formula/watson.rb]: Should this be `Command-line tool` rather than `Command-line too`?

File: Library/Formula/watson.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  end
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    %w[arrow click pytest-runner python-dateutil requests six].each do |r|
```
Comment (DomT4) [inline Library/Formula/watson.rb]: A bit cleaner to do:
```ruby
resources.each do |r|
  r.stage do
```
etc. Only really need the arrays if something needs handling separately for some reason.

**Comment by jmaupetit** [comment]
Thank you again for this review. As you might have already guessed, I used [homebrew-pypi-poet](https://pypi.python.org/pypi/homebrew-pypi-poet) as a template for this formula.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/ff207dc36a4df5ec6bcbffba0a851931b3107f3d. Thank you for your contribution to Homebrew @jmaupetit; we appreciate it! :smiley_cat:.

> I used homebrew-pypi-poet as a template for this formula.

If you're interested in doing so, Tim _may_ be interested in a PR to tweak that on the `poet` GitHub.

**Comment by jmaupetit** [comment]
Thank you for this! I'll take a look at `poet`.

**Pull request closed without merging**